main.go

```diff
@@ -35,16 +41,61 @@ func main() {
 	fmt.Println()
 	log.Infof("Running test")
 
-	testCmd := command.New("flutter", append([]string{"test"}, additionalParams...)...).
+	pr, pw := io.Pipe()
+
+	if _, err := exec.LookPath("tojunit"); err != nil {
+		log.Infof("Command `tojunit` not found, installing...")
+		junitInstallCmd := command.New("flutter", append([]string{"pub", "global", "activate", "junitreport"})...).
```

Comment:
Please print the command before running. 

```
fmt.Println()
log.Donef(fmt.Sprintf("$ %s", junitInstallCmd.PrintableCmdArgs()))
fmt.Println()
```

Reply:
done